gnosis/eth/clients/ens_client.py

```diff
@@ -14,15 +14,33 @@ class EnsClient:
     Resolves Ethereum Name Service domains using ``thegraph`` API
     """
 
-    def __init__(self, network_id: int):
-        self.ethereum_network = EthereumNetwork(network_id)
-        if network_id == self.ethereum_network.SEPOLIA:
-            url = (
-                "https://api.studio.thegraph.com/proxy/49574/enssepolia/version/latest/"
-            )
-        else:  # Fallback to mainnet
-            url = "https://api.thegraph.com/subgraphs/name/ensdomains/ens/"
-        self.url = url
+    class Config:
```

Comment:
Makes sense. Actually started with it but had some warning regarding some keyword parameters. I've added it again since it clearer with a Dataclass.

[d96da58](https://github.com/safe-global/safe-eth-py/pull/1187/commits/d96da58bbf4d30440cec9381419dce3ae66164dc)